config/github_example.yml

```diff
@@ -1,5 +1,5 @@
-# generate the token here: https://github.com/settings/tokens/new
-
-token:
-  repos: ["SUSE/connect", "SUSE/happy-customer", "SUSE/reviewlette", "SUSE/scc-docker-images",
-    "SUSE/scc-provisioning", "SUSE/scc-token", "SUSE/TeamDashboard" ]
+token: # generate the token here: https://github.com/settings/tokens/new
+repos:
+  # - user/repo1
```

Comment:
It is an example file. Why would you comment out those lines here? It's a little bit misleading, I think...